com.endlessm.EknServices.json

```diff
@@ -1,14 +1,13 @@
 {
-    "app-id": "com.endlessm.EknServices",
+    "app-id": "com.endlessm.EknServices2",
+    "branch": "master",
     "runtime": "com.endlessm.apps.Platform",
-    "runtime-version": "1",
+    "runtime-version": "master",
```

Comment:
Should this be "master" or "2" ?

Reply:
I'm thinking it should actually be "stable", since we also have to rename the app ID to EknServices2, but I wanted to make it "master" on the master branch and then change it to "stable" when we branch for the stable version.

Reply:
I think `master` is good for this; then we can open a new PR for renaming it when we release.